apps/alerts/test/cache/store_test.exs

```diff
@@ -84,6 +84,9 @@ defmodule Alerts.Cache.StoreTest do
     alert4 = %Alerts.Alert{id: "126", effect: :cancellation}
 
     Store.update([alert1, alert2, alert3, alert4], nil)
-    assert Store.alert_ids_for_stop_id("543") == ["124", "123"]
+
+    for id <- ["123", "124"] do
+      assert Enum.member?(Store.alert_ids_for_stop_id("543"), id)
+    end
```

Comment:
Yeah, I was thinking about it just in terms of checking that those two values are present rather than that the two arrays are exactly the same. But, easy to change.